.github/workflows/testing_and_prospector.yml

```diff
@@ -0,0 +1,27 @@
+name: Testing Suite
+
+on:
+  push:
+    branches: [ '**' ]
+  pull_request:
+    branches: [ '**' ]
+
+jobs:
+  run-pytest:
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Bring up docker images
+      run: docker-compose up -d
+    - name: Run test suite
+      run: docker-compose run snowshupostgres pytest
+  run-prospector:
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Bring up docker images
+      run: docker-compose up -d
+    - name: Run test suite
+      run: docker-compose run snowshupostgres prospector
```

Comment:
nitt: maybe we want the -M flag so the failure artifacts will be easy to read?

Reply:
I think the -M flag only leaves off the summary block without changing the error message formatting.  I included it here to see what was actually run for CI after the fact. 

The included `test` function in the `entry_point.sh` script that can be run locally runs the prospector with the -M flag so that during development the summary block isn't shown.